### DISCUSSION TITLE
mssql setup and node ace invoke @adonisjs/lucid

### DISCUSSION THREAD

**Discussion by CryusFrey** [adonisjs/core#1685] [Help]
Hello everyone

I'm trying to setup an adonis 5 api project but i'm having some problems.
when running:

```cmd
node ace invoke @adonisjs/lucid
```

the cli gives me back this messagge

```bash
warn Unable to generate manifest file. Check the following error stack for more info
error Make sure to compile the code before running "node ace generate:manifest"
```

I'm also trying to connect to a sql server database and i dunno if this databse.ts object I made is correct

```javascript
connections: {
    mssql: {
      client: "mssql",
      connection: {
        server: Env.get("DB_HOST", "someip") as string,
        port: Number(Env.get("DB_PORT", 1433)),
        user: Env.get("DB_USER", "someuser") as string,
        password: Env.get("DB_PASSWORD", "somepassword") as string,
        database: Env.get("DB_DATABASE", "somedatabase") as string,
      },
    },
  },
```

because when I try to call a store procedure from the db via a controller the error message that comes up is this

```http
Failed to connect to xx.xx.xx.xx:1433 - Cannot call write after a stream was destroyed
```

Me and some colleagues have tried for test to create a manual connection to a mySql db of another project and is working.

As per documentation both npm and node are updated

**Comment by dawidow** [comment]
Did you install it properly? 
Firstly install it with `npm i @adonisjs/lucid@alpha` command. Then go to your .env file and fill all Database fields like DB_USER, DB_PASSWORD, DB_NAME. Then if all is done try to `node ace invoke @adonisjs/lucid`. You do not need to change the database.ts file at all.

**Reply by CryusFrey** [reply]
Still nothing. When I run `node ace invoke @adonisjs/lucid` I get again the aforementioned error and all the data I set on the .env file gets replaced by lucid placeholders

**Comment by thetutlage** [comment]
First things first. Please format your code blocks properly. Here's the link to the markdown guide https://guides.github.com/features/mastering-markdown/

**Reply by CryusFrey** [reply]
thanks for the link. I've updated the discussion

**Reply by thetutlage** [reply]
Do you have these environment variables set inside the `.env` file?
